src/api/test/models/event_test.rb

```diff
@@ -162,7 +162,8 @@ def groups_for_event(e)
 
     assert_equal "Request #{myid} requires review (submit Apache/BranchPack)", email.subject
     # fred is maintainer of the package, hidden_homer of the project, Iggy triggers the event, so doesn't get email
-    assert_equal %w(fred@feuerstein.de homer@nospam.net), email.to.sort
+    # adrian is a member of test_group_b which is a maintainer of the package
+    assert_equal %w(adrian@example.com fred@feuerstein.de homer@nospam.net), email.to.sort
```

Comment:
Why does this differ now?

Reply:
Because in the test fixtures the user with login `adrian_downloader` and email `adrian@example.com`  is a member of `test_group_b` which is a maintainer of the `TestPack` package in this example. Before if you were  a member of a group with no email and the group was supposed to receive a notification then you as a user would not get a notification. This seems wrong to me and is not in line with the rest of the logic for this so I think its a mistake.